mix.exs

```diff
@@ -9,7 +9,7 @@ defmodule Neuron.Mixfile do
       app: :neuron,
       description: "A GraphQL client for elixir.",
       version: @version,
-      elixir: "~> 1.7",
+      elixir: "~> 1.13",
       build_embedded: Mix.env() == :prod,
       start_permanent: Mix.env() == :prod,
       test_coverage: [tool: Coverex.Task],
```

Comment:
I'm unsure whether you want to retain Coverex, https://github.com/alfert/coverex, which have not been updated for 6 years.